src/main/java/io/leonis/zosma/ipc/controllers/ControllerSetPublisher.java

```diff
@@ -0,0 +1,52 @@
+package io.leonis.zosma.ipc.controllers;
+
+import com.studiohartman.jamepad.*;
+import io.reactivex.Flowable;
+import java.util.Set;
+import java.util.concurrent.TimeUnit;
+import java.util.stream.*;
+import org.reactivestreams.*;
+
+/**
+ * The Class ControllerSetPublisher.
+ *
+ * Publishes all <b>connected</b> controllers at a provided interval.
```

Comment:
why the `<b>`? if you really want to stress the importance i guess use `<strong>` but i think it's unnecessary